sql/core/src/main/scala/org/apache/spark/sql/execution/command/commands.scala

```diff
@@ -87,6 +87,51 @@ case class ExecutedCommandExec(cmd: RunnableCommand) extends LeafExecNode {
   }
 }
```

Comment:
I think the `metrics` in `RunnableCommand` is mainly for use in `DataWritingCommand`. Now `DataWritingCommand` is a separate class other than `RunnableCommand`, do we still need `metrics` in `RunnableCommand`?

Reply:
Well, I prefer to keep it. It is used only in `ExecutedCommandExec`, and in the future if there are new `RunnableCommand` with metrics, the new command can just override `metrics`.